src/dist/replication/lib/bulk_load/replica_bulk_loader.h

```diff
@@ -33,12 +33,18 @@ class replica_bulk_loader : replica_base
                             const std::string &cluster_name,
                             const std::string &provider_name);
 
+    // compare meta bulk load status and local bulk load status
+    // \return ERR_INVALID_STATE if it is invalid for replica whose local status is `local_status`
+    // while meta status is `meta_status`
```

Comment:
I really don't know what does this sentence mean :(

Reply:
oh, I have updated the comment.